### PR TITLE
Now runs with new networkx

### PR DESCRIPTION
**Pull request by Lamaun**
The change of networkx graph data structure broke this package.

### CODE REVIEW

File: nxmetis/tests/test_metis.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def setUp(self):
         self.node_list = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j',
                           1, 2, 3, 4, 5, 6]
         self.G = nx.Graph()
-        self.G.add_path(self.node_list)
+        self.G.update(nx.path_graph(self.node_list))
```
Comment (dschult) [inline nxmetis/tests/test_metis.py]: ```self.G.add_path(stuff)``` -- > ```nx.add_path(self.G, stuff)``` is a good way to add a path.

**Comment by dschult** [comment]
This repository hasn't been actively updated in a while.
Probably the python versions tested on Travis should match the current networkx versions.
It is possible that other deprecated functionality from NetworkX could cause problems too. 
Thanks for doing this!

**Comment by Lamaun** [comment]
I haven't worked with travis yet, should we replace travis.yml with something similar to https://github.com/networkx/networkx/blob/master/.travis.yml?

**Comment by dschult** [comment]
Can you try a more simple change first?  I'm hoping it doesn't require the more heavy lifting.

Change it to python versions 3.6, 3.7, 3.8, pypy3 and let's see what Travis says.
Thanks!

**Comment by Lamaun** [comment]
Seems like that works

**Comment by dschult** [comment]
Excellent -- anything else before we merge?

**Comment by Lamaun** [comment]
Not from my side. Could we get this into pypi?

**Comment by dschult** [comment]
I'll look into getting it into pypi.

**Pull request merged**